vms/vms/static/vms/js/hide_resume.js

```diff
@@ -0,0 +1,9 @@
+$(document).ready(function() {
+    $("input[type='file']").change(function() {
+    var FileLength = $("#id_vol-resume_file")[0].files.length;
+    if (FileLength!==0){
```

Comment:
better use http://api.jquery.com/toggle/?

Reply:
Yes toggle is better to use, implemented this too. Have a look.